packages/venia-ui/package.json

```diff
@@ -75,6 +75,8 @@
   "peerDependencies": {
     "@magento/babel-preset-peregrine": "~1.0.0",
     "@magento/peregrine": "~4.0.0",
+    "@magento/pwa-buildpack": "~4.0.0",
+    "@magento/venia-concept": "~4.0.0",
```

Comment:
The idea is that `peregrine` is used by `venia-ui` is used by `venia-concept`. This seems like a red flag to me? But maybe I'm misunderstanding peer deps :P

I think this, along with the imports of `venia-concept` files in `venia-ui` just seem like red flags to me in terms of where we are supposed to be keeping these things. Maybe in the future we will move the majority of the base config to the top-level and `venia-concept` and `venia-ui/storybook` can just import/use it.

TL;DR - I think I'm fine with this, just rambling.

Reply:
Yeah I think for the storefront app that's correct. For Storybook we relaxed the rules a bit so we could run it easily.

Reply:
Yeah, it's a tough call. I do think @sirugh has the right of it here.

It's true that `venia-ui` components are hard to render without the glue that `venia-concept` adds, so we need to get that glue from somewhere. But `venia-ui` is built to be a dependency of `venia-concept`, so I'd prefer that it didn't list `venia-concept` as a dependency or import anything from it. Thinking about our various options here:

Option | Evaluation
:--- | :---:
Duplicate `store.js`, `webpack.config.js`, and `index.css` | bad
Move these files up to some shared package | bad
Import these files from `venia-concept` | not ideal
Move `venia-ui` storybook stories into `venia-concept` | okay?

Overall, I would accept the approach in this PR because the work's been put in, it has value, and only the Storybook files are importing from ` venia-concept`. But eventually I want to remove the circular dependency.